src/drivers/rc_input/RCInput.cpp

```diff
@@ -210,6 +211,10 @@ RCInput::fill_rc_in(uint16_t raw_rc_count_local,
 			_rc_in.rssi = 255;
 		}
 
+#else
+		_rc_in.rssi = 255;
```

Comment:
https://github.com/PX4/Firmware/blob/ea3d2023700b8211dc12e25029e8d8c6725c5218/msg/input_rc.msg#L25

We should either update `input_rc` or the usage here to be consistent.

Reply:
@dagar, done.
there different types of the rssi strength trough the system, there int32, uint16, and int8.
do you think it ok to go over them and change them all to int8?
and then undefined will be changed to -1

Reply:
At the moment I can't imagine why it wouldn't be acceptable to normalize all cases like this, but to be paranoid I'd do a quick review of all the supported protocols in the RC library (https://github.com/PX4/Firmware/tree/master/src/lib/rc).

Reply:
anyway, uint8 will better choice, because uint seems the be the standard on Mavlink:

https://github.com/mavlink/c_library_v2/blob/1eff57ee1e54fbed0b24617b1d4452d26d8b2221/message_definitions/common.xml#L4463